# Документация по файлу `errors.py`


## `PrototypeError.__init__`<br>
Исключение, возникающее при ошибке в прототипах.<br>
**Args:**<br>
file_path (str): Путь к файлу с ошибкой.<br>
location (str): Место в файле, где возникла ошибка.<br>
message (str): Описание ошибки.<br>
<br>
